### ISSUE TITLE
import/no-cycle not triggered when using paths relative to the project root

### ISSUE THREAD

**Issue by SimonMueller** [import-js/eslint-plugin-import#2468]
When using imports like "src/services/module.a" in a typescript project with an import resolver instead of "../module.a" cycles are not detected by the rule "import/no-cycle". The imports are correctly resolved and only the error from the cycle is missing in the eslint output.

When using relative imports like "../module.a" the cycle is correctly detected.
The same behaviour is observed when using plain js files without  the typescript eslint parser.

The behaviour can be reproduced with the following two files and the configuration listed below:

module A
```
import { ModuleB } from 'src/services/module.b';

export class ModuleA {}
``` 

module B
```
import { ModuleA } from 'src/services/module.a';

export class ModuleB {}
```

tsconfig.json
```
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "es2020",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "skipLibCheck": true,
    "moduleResolution": "node",
    "esModuleInterop": true
  },
  "include": ["src/**/*"]
}

```

.eslintrc.js
```
module.exports = {
  plugins: ["@typescript-eslint/eslint-plugin", "import"],
  extends: [
    "plugin:@typescript-eslint/recommended","plugin:import/recommended",
  ],
  settings: {
    "import/resolver": {
      node: {
        paths: ["."],
        "extensions": [".ts"]
      },
    },
  },
  root: true,
  parser: "@typescript-eslint/parser",
  parserOptions: {
    "sourceType": "module",
    "project": ["./tsconfig.json"]
  },
  rules: {
    "import/no-cycle": "error",
    "@typescript-eslint/no-unused-vars": ["off"]
  }
};
```

versions from package.json 
```
    "@typescript-eslint/eslint-plugin": "^5.23.0",
    "@typescript-eslint/parser": "^5.23.0",
    "eslint": "^8.2.0",
    "eslint-plugin-import": "2.26.0",
```

**Comment by JounQin** [comment]
close in favor of https://github.com/import-js/eslint-import-resolver-typescript
